control-plane/agents/src/bin/core/main.rs

```diff
@@ -141,10 +145,12 @@ async fn main() {
     let cli_args = CliArgs::args();
     utils::print_package_info!();
     println!("Using options: {cli_args:?}");
-    utils::tracing_telemetry::init_tracing(
+    utils::tracing_telemetry::init_tracing_with_eventing(
         "core-agent",
         cli_args.tracing_tags.clone(),
         cli_args.jaeger.clone(),
+        "agent-core",
```

Comment:
Why is there core-agent and agent-core twice?

Reply:
"core-agent" is used for service name for tracing. 

Passing the source component "agent-core" from here. There was a review comment to use the source_component "agent-core" in from_str trait, because the app label is "agent-core" for the core-agent

Reply:
Can we use the same argument, we can rename core-agent to agent-core